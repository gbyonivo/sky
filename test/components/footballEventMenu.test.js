import React from 'react';
import { createStore } from 'redux';
import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import combinedReducer from '../../src/reducers';
import FootballEventsMenu from '../../src/components/footballEventsMenu';

describe('FootballEventsMenu', () => {
  it('should render correctly', () => {
    const store = createStore(combinedReducer(), { footballEventsReducer: { footballEvents: [] } });
    const actual = renderer.create(<Provider store={store}><FootballEventsMenu/></Provider>).toJSON();
    expect(actual).toMatchSnapshot();
  });
});