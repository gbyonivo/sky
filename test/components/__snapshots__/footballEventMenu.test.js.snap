// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FootballEventsMenu should render correctly 1`] = `
<div
  className="
      footballEventsMenu
      isExpanded
      onlyListOnScreen"
>
  <h2
    className="footballEventsMenuHeader"
  >
    Football Events
  </h2>
  <div
    className="priceFormatToggle"
  >
    <div
      className="text"
      style={
        Object {
          "color": "#ff6464",
        }
      }
    >
      Fraction Price
    </div>
    <div
      className="toggle switch on"
      onClick={[Function]}
    >
      <div
        className="ball left"
      />
    </div>
    <div
      className="text"
      style={
        Object {
          "color": "#36d836",
        }
      }
    >
      Decimal Price
    </div>
  </div>
  <div
    className="showPrimaryMarketToggle"
  >
    <div
      className="text"
    >
      Display Primary Market
    </div>
    <div
      className="toggle switch off"
      onClick={[Function]}
    >
      <div
        className="ball right"
      />
    </div>
  </div>
  <div
    className="footballEventsGroups"
  />
</div>
`;
